.gitignore

```diff
@@ -7,4 +7,3 @@
 /build
 /captures
 .externalNativeBuild
-app/

```

Comment:
I'm pretty sure we need this. There is a lot of generated code that gets added that we don't need.

Reply:
Isn't the app folder the one that contains the rest of the code?